dataset_doi.yaml

```diff
@@ -26,7 +26,7 @@ epacamd_eia:
   production_doi: 10.5281/zenodo.6633769
   sandbox_doi: 10.5072/zenodo.3136
 epacems:
-  production_doi: 10.5281/zenodo.4127054
+  production_doi: 10.5281/zenodo.10233185
```

Comment:
this was an old-ish requried change. ella had to make a whole new cems archive on the transition from the year/state annual files to the quarterly files but probably forgot to update this guy.